catalog/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
 
-# Create your views here.
+from catalog.models import Book, Author, BookInstance, Genre
+def index(request):
+  """View function for home page of site."""
+  num_books = Book.objects.all().count()
```

Comment:
loại bỏ code dư thừa nhen
```suggestion
  num_books = Book.objects.count()
```